### ISSUE TITLE
Issue with "Firefox user agent string reference": Firefox iOS string for iPad

### ISSUE THREAD

**Issue by karlcow** [mdn/content#4779]
MDN URL: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent/Firefox

#### What information was incorrect, unhelpful, or incomplete?

Currently the pattern for the iPad is:
```
Mozilla/5.0 (iPad; CPU iPhone OS 8_3 like Mac OS X) AppleWebKit/600.1.4 (KHTML, like Gecko) FxiOS/1.0 Mobile/12F69 Safari/600.1.4
````

#### Specific section or headline?
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent/Firefox#firefox_for_ios

#### What did you expect to see?

```
Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_4) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/13.1 Safari/605.1.15
```

#### Did you test this? If so, how?
see https://github.com/mozilla-mobile/firefox-ios/blob/main/Shared/UserAgent.swift#L52

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/http/headers/user-agent/firefox`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent/Firefox
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/http/headers/user-agent/firefox/index.html
* Last commit: https://github.com/mdn/content/commit/8d4fd702d9668604cb2788da5bd9fc9b796ec30a
* Document last modified: 2021-03-17T20:27:56.000Z

</details>

**Comment by hamishwillee** [comment]
@karlcow Can you please verify the fix in #8660

Are you really saying that the UA string for iPad (iOS) is the same as for mac desktop (iOS)? Why then would the iPhone and iPod not follow the same pattern? Or to put it another way, does that not make these other statements incorrect too - in particular the one below the heading?

I'm not saying you're wrong - it just seems inconsistent enough to be worth questioning/testing.

![image](https://user-images.githubusercontent.com/5368500/132144107-a59db71d-4087-44fa-b5a1-e8daedd0f1e1.png)

**Comment by karlcow** [comment]
sorry late to the part, I was off for a while. 👍
